### ISSUE TITLE
Broken symbolic icons on file manager

### ISSUE THREAD

**Issue by eugenialoli** [SylEleuth/gruvbox-plus-icon-pack#97]
I just installed Linux Mint on a new PC this afternoon. I then used the zip file from the current git (21 Oct 2024).
The Nemo file manager doesn't find the symbolic icons on its toolbars. They all show up as not-there.
For comparison, I put in front of the screen my macbook air, with linux mint using the icons from July 31 release. These work fine. Only the git master on the back screen doesn't.

Additionally, both with the July 31st release and git master, the following icons aren't getting applied (the apps are installed via the official mint repo as .deb): Dia, Gnumeric, Foliate, Ltris, Liferea, Supertux.

![PXL_20241021_154948356](https://github.com/user-attachments/assets/d094c239-4905-4437-8696-74e75b7a11c3)

**Comment by SylEleuth** [comment]
Do not `download zip`. Clone repository and then tell me if it works.

**Comment by SylEleuth** [comment]
I have a very weird behaviour.

What I did:
1. Cloned repo and linked it to the `icons` folder
2. Checked `Nemo` app - missing icons
3. Renamed `actions` folder to anything else
4. Checked `Nemo` app - fallback icons working 
5. Renamed `actions` folder back to original name
6. Checked `Nemo` app - everything is fine now.

Can You please do the same?

**Comment by eugenialoli** [comment]
I was able to reproduce it. Even when I logout and log back in the icons are retained on Nemo (just in case was a residual from the previously loaded themes). Weird behavior indeed. Other apps have this problem too, like the Software Manager. Basically, everything that's Mint-specific.

BTW, last night I replaced the new git index.theme with the Jul 31 one, and that seemed to fix it too. Not sure if the bug is in the new index.theme, or in Mint.
